docs/adr/adr-010-incentivized-testnet-monitoring.md

```diff
@@ -167,14 +168,44 @@ The Prometheus docs state the following with regard to [Denial of Service](https
+#### Scenario A: Node operators
+
+Pros
+
+- This deployment architecture is more representative of mainnet where node operators will run their own telemetry stack to monitor their node. Exposing node operators to OTEL Collector during incentivized testnet allows them to practice this deployment architecture prior to mainnet.
+- Node operators will have an "incentive" to maintain high uptime for their OTEL Collector.
+
+Cons
+
+- Additional operational burden for incentivized testnet participants. We can mitigate this concern by providing easy install steps and scripts.
+
+#### Scenario B: Celestia team
+
+Pros
+
+- It will be easier for nodes to participate if they only have to deploy one piece of software (celestia-node) and not two (celestia-node and OTEL Collector).
+
+Cons
+
+- Node operators will lose the ability to monitor their own celestia-node. Since opentelemetry-go supports configuring only one exporter ( [open-telemetry/opentelemetry-go#3055](https://github.com/open-telemetry/opentelemetry-go/issues/3055)), if node operators were obligated to export metrics to a Celestia team managed OTEL Collector endpoint, they wouldn't be able to export to their own OTEL Collector (and by proxy any telemetry platforms they wish to use). This violates the desideratum:
 
     > We would like to make it possible for node operators to monitor their own nodes with existing telemetry tools (e.g. Grafana and Uptrace)
 
-1. Node operators will have an "incentive" to maintain high uptime for their OTEL Collector. If the Celestia team took on this responsibility and failed to provide a highly available solution, then node operators would be penalized for downtime of a component they have no control over.
-1. We expect 1500+ node operators during the incentivized testnet and there is minimal documentation on the scale of workload an individual OTEL Collector can handle. We'd have to design and operate a highly available OTEL Collector fleet to maintain high uptime for node operators. At this time no cloud managed offerings for OTEL Collector exist.
+- If the Celestia team took on this responsibility and failed to provide a highly available solution, then node operators would be penalized for downtime of a component they have no control over.
+- We expect 1500+ node operators during the incentivized testnet and there is minimal documentation on the scale of workload an individual OTEL Collector can handle. We'd have to design and operate a best-effort highly available OTEL Collector fleet to maintain high uptime for node operators. At this time no cloud managed offerings for OTEL Collector exist.
```

Comment:
If we rely on AWS, we should consider [this](https://aws-otel.github.io/docs/introduction). I am pretty sure it has a horizontal autoscaling feature that AWS provides(cc @sysrex).

Reply:
As far as I can tell, https://aws.amazon.com/otel/faqs/ doesn't provide any additional functionality that we need. We don't plan on using AWS X-Ray, Amazon CloudWatch, or Amazon Managed Service for Prometheus at this time.

@Wondertan what component in the AWS distro do you think we should use?

AWS has several auto-scaling features depending on the service / infrastructure we're using. If we intend on hosting a highly available OTEL Collector fleet then yes we should leverage a cloud provider's load balancers / autoscaling.

Reply:
Resolving b/c no actionable changes needed in ADR